libs/common/src/platform/services/noop-notifications.service.ts

```diff
@@ -0,0 +1,28 @@
+import { NotificationsService as NotificationsServiceAbstraction } from "../../abstractions/notifications.service";
+import { LogService } from "../abstractions/log.service";
+
+export class NoopNotificationsService implements NotificationsServiceAbstraction {
+  constructor(private logService: LogService) {
+    logService.debug("NoopNotificationsService");
+  }
+
+  init(): Promise<void> {
+    this.logService.info("Initializing notification service");
```

Comment:
This statement is a bit nondescript, I think we should note that you won't receive any updates.

```suggestion
    this.logService.info("Initializing no-op notification service, no push notifications will be received");
```